### ISSUE TITLE
NOD | Add Part III, box 11 pages to online form

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#42493]
## Value statement

**_As a Veteran_** filling out an online Notice of Disagreement form
**_I want to_** be able to request an extension, or appeal a VHA denial, which is available on the paper form
**_So that_** I can fill out the online form instead of filling out a paper form

## Background Context
**Need stakeholder approval before implementation.** 

The [Notice of Disagreement paper form](https://www.va.gov/vaforms/va/pdf/VA10182.pdf) includes two checkboxes in part 3, box 11 which allow the Veteran to 1. request an extension, and 2. appeal a VHA denial specifically; and add additional comments to an attachment.

([Lighthouse API v2 schema](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/v2/10182.json)) has already been updated to include 3 fields:

- Extension request (`true` or `false`; yes or no radio)
- Extension reason (string with max length of 2300; text box)
- Appeal of VHA denial (`true` or `false`; yes or no radio)

And the schema has switched the submission data from using `countryName` to `countryCodeISO2` 

## Acceptance Criteria
- [ ] Users are able to select and move forward with issues that have a decision date that's over a year old.
- [x] Users are able to select that they want an extension
- [x] Users are able to enter a reason for their extension
- [x] Users are able to indicate that the appeal is a for a VHA denial
- [x] Users are able to see and make edits on the Review & Submit screen
- [ ] Unit tests complete (90% coverage)
- [ ] E2E tests complete
- [ ] Accessibility testing complete
- [ ] Reviewed and approved by product and/or design

## Designs and Build Notes
- [Design](https://www.sketch.com/s/0424c51d-7818-43ea-8821-6de91eb7ab27/v/PJ1ZDv/p/6863DCFF-CAA2-4BAD-BE68-F2D584466FFB/canvas?posX=-4001.220458984375&posY=-699.7330932617188&zoom=0.3273887634277344)
- [Review and Submit](https://www.sketch.com/s/0424c51d-7818-43ea-8821-6de91eb7ab27/a/bLAnb5V)
- [Content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Notice-of-Disagreement/design/sourceofcontenttruth.md)

## Out of scope
- Good cause examples
- Document upload feature

## Tasks
- [ ] Get requirements approved by BVA (John Gosnell)
- [ ] Add extension request & reason page
- [ ] Add VHA denial page
- [ ] Add migration code
- [ ] Update submit transformer
- [ ] Update & add unit tests
- [ ] Update mock data

## Definition of Ready (user story)
- [ ] _Clear value description_
- [ ] _Testable acceptance criteria_
- [ ] _Accessibility added to acceptance criteria_
- [x] _Approved designs attached_
- [ ] _Sample data provided where appropriate_
- [ ] _Estimated to fit within the sprint_
- [ ] _Dependencies and blockers linked_

## Definition of Done (user story) 

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Reviewed and approved by product and/or design
---

**Comment by eileen-coforma** [comment]
Questions:  
- What is considered 'good cause' for extension? I can't find any more information about it.
- What does it mean to appeal a VHA denial? Is the process any different for veterans? That information is not part of the form.

**Comment by Mottie** [comment]
Unassigned Eileen, this is a frontend ticket.

**Comment by data-doge** [comment]
@saderagsdale Speaking just for myself, I think this would be an 8 because it would be my first real foray into FE dev on the VA platform.

**Comment by saderagsdale** [comment]
@eileen-coforma Can you add the design links to this ticket for the FE work?

**Comment by eileen-coforma** [comment]
Emailed John for approval. Moving it to blocked until we get a response. Will follow-up again tomorrow if we don't get a response by EOD.

**Comment by eileen-coforma** [comment]
@Mottie We are good to move for the pages. The content may have changes (still waiting for feedback from the intake team), but the pages and functionality are good to go.

**Comment by Mottie** [comment]
Ready for testing in staging - submissions may fail until the backend is ready

**Comment by eileen-coforma** [comment]
@Mottie The only thing missing is the new AC I added. Contentions need to be unblocked, so users can select ones that don't fall within the 1 year deadline.

**Comment by Mottie** [comment]
It's done on the frontend... I'll look into this tomorrow.

**Comment by Mottie** [comment]
Please test this in staging. Issues older than 1 year will no longer be filtered out when the Part III, box 11 questions are visible via the feature flag.

**Comment by eileen-coforma** [comment]
Confirmed that the issues are now showing up (no longer limited to 1 year)

**Comment by Mottie** [comment]
[Opened a PR to use the `v1` submission endpoint ](https://github.com/department-of-veterans-affairs/vets-website/pull/25043)- we should be able to fully test these changes in staging later today
